manifests/config.pp

```diff
@@ -1,58 +1,68 @@
+# Class: check_mk::config
+#
+# Configures check_mk.
+# 
+# @summary Configures check_mk. Should not be called directly.
+# 
```

Comment:
`@api private` instead of the comment `Should not be called directly`.  You can probably lose lines 1-4 too.

Reply:
Implemented in all the files I changed.